beacon_node/beacon_chain/src/builder.rs

```diff
@@ -1010,25 +976,23 @@ where
                 &head.beacon_state,
             ) {
                 crit!(
-                    log,
-                    "Weak subjectivity checkpoint verification failed on startup!";
-                    "head_block_root" => format!("{}", head.beacon_block_root),
-                    "head_slot" => format!("{}", head.beacon_block.slot()),
-                    "finalized_epoch" => format!("{}", head.beacon_state.finalized_checkpoint().epoch),
-                    "wss_checkpoint_epoch" => format!("{}", wss_checkpoint.epoch),
-                    "error" => format!("{:?}", e),
+                    head_block_root =%head.beacon_block_root,
+                    head_slot =%head.beacon_block.slot(),
+                    finalized_epoch =%head.beacon_state.finalized_checkpoint().epoch,
+                    wss_checkpoint_epoch =%wss_checkpoint.epoch,
```

Comment:
You know what's coming :smiling_face_with_tear: 
This also applies to the log lines above and below
```suggestion
                    wss_checkpoint_epoch = %wss_checkpoint.epoch,
```